x-pack/plugin/async-search/build.gradle

```diff
@@ -10,8 +13,6 @@ base {
   archivesName = 'x-pack-async-search'
 }
 
-addQaCheckDependencies(project)
```

Comment:
This is no longer necessary because the `:qa` child project has been removed. This is used to ensure that if I run `./gradlew :x-pack:plugin:async-search:check` that the check tasks for all child projects are _also_ run.